test/fixtures/esm-common-chunk/a.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-console */
+Promise.all([
+  System.import('./b'),
+  System.import('./c'),
```

Comment:
I would update these to be just `import('./b')` style since the `System.import` version is pretty much deprecated.

Reply:
This only works with babel-eslint in the mix at the moment. 

We can either attack the babel issues that lead to it's removal or stick with System.import for the time being.

Reply:
Oh really ... I didn't know that. Leaving it is just fine with me.